test/example/test.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+import 'test_foo.dart';
+
+void main() {
+  const Duration delay = Duration(seconds: 4);
+  Function work;
```

Comment:
this is only assigned once, should it be written as a function instead of assigning a lambda?

https://www.dartlang.org/guides/language/effective-dart/usage#do-use-a-function-declaration-to-bind-a-function-to-a-name

Reply:
I use this script for manual testing of the debugger, so it's got some oddities. I can look at using a class instead of top-level functions in order to get the same behavior I need.

Reply:
The function does not need to be top level, it can be defined within `main`. From what I can see you can use exactly  what you have if you:

- Remove variable declaration
- Remove the ` = ` at the definition point
- Remove the semicolon following the last `}`

Reply:
Oh, when you were testing did you specifically need a variable that held a closure? If that's the case maybe add a comment?